src/main/java/org/folio/service/orders/flows/update/open/OpenCompositeOrderFlowValidator.java

```diff
@@ -159,4 +166,16 @@ private Future<Void> validateLocationRestrictions(CompositePoLine poLine, List<F
     return Future.failedFuture(new HttpException(422, ErrorCodes.FUND_LOCATION_RESTRICTION_VIOLATION, parameters));
   }
 
+  private List<String> getPermanentLocationIdFromHoldings(List<String> holdingIds, RequestContext requestContext) {
+    List<String> permanentLocationIds = new ArrayList<>();
```

Comment:
I am not sure does InventoryManager has check that when holdingIds are empty - to not make call to mod-inventory, probably better to add this check here also

Reply:
Thanks, added check